src/service/activity-impl.js

```diff
@@ -204,11 +209,12 @@ export class Activity {
   /** @private */
   setUpActivityListeners_() {
     for (let i = 0; i < ACTIVE_EVENT_TYPES.length; i++) {
-      this.unlistenFuncs_.push(listen(this.win_.document.documentElement,
+      this.unlistenFuncs_.push(listen(this.win_.document,
         ACTIVE_EVENT_TYPES[i], this.boundHandleActivity_));
     }
 
     this.viewer_.onVisibilityChanged(this.boundHandleVisibilityChange_);
+    this.viewport_.onScroll(this.boundHandleActivity_);
```

Comment:
`viewport.onScroll` doesn't return an unlisten function so it can't be added. But `viewer.onVisibilityChanged` does so I'll add that.